pico_w/freertos/httpd/picow_freertos_httpd.c

```diff
@@ -17,6 +17,8 @@
 
 void httpd_init(void);
 
+static absolute_time_t connect_time;
```

Comment:
Looking at the code, this seems to be "wifi connection time"? I wonder if people might confuse it for "HTTP-request connection time"? :shrug: